jax/experimental/jax2tf/README.md

```diff
@@ -78,9 +78,52 @@ tf.saved_model.save(my_model, '/some/directory')
 restored_model = tf.saved_model.load('/some/directory')
 ```
 
+To explain: 
+
+  * saved model and custom gradients
+  * saved model and XLA
+
```

Comment:
I take it this is a TODO? I think in that case it should be tagged explicitly with the "TODO" keyword (?).
Otherwise LGTM :)

Reply:
Added TODO